etc/profile-a-l/liferea.profile

```diff
@@ -51,3 +51,11 @@ tracelog
 disable-mnt
 private-dev
 private-tmp
+
+dbus-user filter
+dbus-user.own net.sourceforge.liferea
+dbus-user.talk ca.desrt.dconf
+dbus-user.talk org.freedesktop.Notifications
+dbus-user.talk org.freedesktop.secrets
+dbus-user.talk org.gnome.Shell.Notifications
```

Comment:
`org.gnome.Shell.Notifications`? `org.freedesktop.Notifications` should be enough, it don't?

Reply:
The legend in https://github.com/netblue30/firejail/wiki/Restrict-D-Bus confused me here. It has a note that org.freedesktop.Notifications is ￼ for GNOME >= 3.36.1 and I wanted to make sure liferea notifications work regardless of GNOME version. Let me do a quick test without `dbus-user.talk org.gnome.Shell.Notifications`

Reply:
@rusty-snake org.gnome.Shell.Notifications is indeed unneeded. I also opted to tighten the dbus filtering and adding comments. What do you think?